content/docs/user-guide/data-management/remote-storage/amazon-s3.md

```diff
@@ -0,0 +1,244 @@
+[env vars](#environment-variables).
+
+[aws cli config]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+<admon type="info">
+
+The AWS user needs the following permissions: `s3:ListBucket`, `s3:GetObject`,
+`s3:PutObject`, `s3:DeleteObject`.
+
+</admon>
+
+[s3]: https://aws.amazon.com/s3/
+[existing s3 bucket]:
+  https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html
+[folder key]:
+  https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html
+
+## Custom authentication
+
+If you don't have a regular local S3 setup or if you want to change the auth
+method for some reason, you can set the following config params with
+`dvc remote modify --local`.
+
+<admon type="warn">
+
+The `--local` flag is needed to write sensitive user info to a Git-ignored
+config file (`.dvc/config.local`) so that no secrets are leaked through Git. See
+`dvc config` for more info.
+
+</admon>
+
+To use custom [AWS CLI config or credential files][aws-cli-config-files], or to
+specify a [profile name], use `configpath`, `credentialpath`, or `profile`:
+
+```cli
+$ dvc remote modify --local myremote \
+                    configpath 'path/to/config'
+# or
+$ dvc remote modify --local myremote \
+                    credentialpath 'path/to/credentials'
+# and (optional)
+$ dvc remote modify --local myremote profile 'myprofile'
+```
+
+[aws-cli-config-files]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+[profile name]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+
+Another option is to use an AWS access key ID (`access_key_id`) and secret
+access key (`secret_access_key`) pair, and if required, an [MFA] session token
+(`session_token`):
+
+```cli
+$ dvc remote modify --local myremote \
+                    access_key_id 'mysecret'
+$ dvc remote modify --local myremote \
+                    secret_access_key 'mysecret'
+$ dvc remote modify --local myremote \
+                    session_token 'mysecret'
+```
+
+[mfa]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa.html
+
+## S3-compatible servers (non-Amazon)
+
+Set the `endpointurl` parameter with the URL to connect to the S3-compatible
+service (e.g. [MinIO], [IBM Cloud Object Storage], etc.). For example, let's set
+up a [DigitalOcean Space] (equivalent to a bucket in S3) called `mystore` found
+in the `nyc3` region:
+
+```cli
+$ dvc remote add -d myremote s3://mystore/path
+$ dvc remote modify myremote endpointurl \
+                    https://nyc3.digitaloceanspaces.com
+```
+
+<admon type="info">
+
+Any other S3 parameter can also be set for S3-compatible storage. Whether
+they're effective depends on each storage platform.
+
+</admon>
+
+[minio]: https://min.io/
+[digitalocean space]: https://www.digitalocean.com/products/spaces
+[ibm cloud object storage]: https://www.ibm.com/cloud/object-storage
+
+## Cloud versioning
```

Comment:
Two thoughts on cloud versioning:

1. It could probably be moved closer to the top (see https://github.com/iterative/dvc.org/pull/4293)
2. You could drop `worktree` in this PR, or we can do it separately since it's being dropped

Reply:
See https://github.com/iterative/dvc.org/pull/4309.

Reply:
Let me figure out another structure now that there's a single CV-related option (prob doesn't deserve an entire H2)... ⏳

Reply:
OK I moved it up (left the H2) and restructured the text to accommodate the change. PTAL @dberenbaum